File: PO/Project/C++/src/sheep.cpp
#include "sheep.h"

sheep::sheep(world* world_point, YX position)
:animal(4,4, position, world_point)
{
    avatar = 'S';
}

organizm* sheep::MakeChild(YX new_position)
{
    return (new sheep(world_point, new_position));
}

sheep::~sheep()
{
}

File: PO/Project/JavaProject/src/Animals/Sheep.java
package Animals;

import Abstractions.*;
import com.google.gson.JsonObject;
import java.awt.*;

public class Sheep extends Animal {

    public Sheep(World world, Point position) {
        super(4, 4, position, world);
        this.avatar = 'S';
        this.image_path = "Sheep.png";
        loadImage();
    }

    @Override
    public Organism makeChild(Point newPosition) {
        return new Sheep(world, newPosition);
    }
}

